### DISCUSSION TITLE
importing pyteomics - missing __init__ file?

### DISCUSSION THREAD

**Discussion by stuarthaze** [levitsky/pyteomics#114] [Q&A] [answered]
I'm new to pyteomics and just installed with pip, however when I try to do a standard import like
`import pyteomics as pt`
I can't access any of the functions using `pt.parser.parse('PEPTIDE')`, but instead get an AttributeError
It seems I have to import each function individually as I need them. I also checked my installation and there's no `__init__.py` file. Does someone know if this intended, or am I doing something wrong?

**Comment by levitsky** [comment, marked as answer]
I think this is how [packages](https://docs.python.org/3/tutorial/modules.html#packages) are supposed to work, in general.
You can import each function or each module, e.g.:

```python
from pyteomics import parser
parser.parse('PEPTIDE')
```

**Reply by stuarthaze** [reply]
Thanks for the reply. I generally import using a reference to the package name to avoid conflicts. Otherwise, if I'm using another package with a parser function it would be cause problems, but it's always possible to import with an alias. It seems like there's lots of useful tools here. Thanks again!

**Reply by levitsky** [reply]
Of course, that's also possible:

```python
import pyteomics.parser
pyteomics.parser.parse('PEPTIDE')
```

I'm glad that you're finding Pyteomics useful!
